### DISCUSSION TITLE
Welcome to the awesome-debrid Discussions!

### DISCUSSION THREAD

**Discussion by yowmamasita** [debridmediamanager/awesome-debrid#6] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by 8hvb4tcksv-sketch** [comment]
Not sure how much this is maintained, but as someone who has been reading up on all this stuff nonstop for the past 2-3 months (after family owned a Kodi builder years ago, me not realizing what it was at the time) I very much appreciate the links and descriptions you've provided in one location, and I honestly mean these specific ones. 
Thank you!
